WordPress/src/test/java/org/wordpress/android/viewmodel/activitylog/ActivityLogViewModelTest.kt

```diff
@@ -35,9 +37,9 @@ class ActivityLogViewModelTest {
     @Mock private lateinit var site: SiteModel
     private val actionCaptor = argumentCaptor<Action<Any>>()
 
-    private var events: MutableList<List<ActivityLogModel>?> = mutableListOf()
+    private var events: MutableList<List<ActivityLogListItemViewModel>?> = mutableListOf()
     private var eventListStatuses: MutableList<ActivityLogListStatus?> = mutableListOf()
-    private val activityLog = listOf<ActivityLogModel>(mock())
+    private val activityLogList = mutableListOf<ActivityLogModel>()
```

Comment:
if you have a mutable list initialized as a field, it will be reused in all the tests and might lead to some unpredictability (you add something to the list in one test, it will still be there in the second one). I'd either keep it immutable or changed it to `lateinit var` and initialized it `setup` function. I'd have `initializeActivityList()` function return a new list and have `activityLogList = initializeActivityList()` in the setup function.